### ISSUE TITLE
Modern Beta compat issue

### ISSUE THREAD

**Issue by NotaScout** [ValkyrienSkies/Valkyrien-Skies-2#716]
### Mod Name

Modern Beta

### This issue occurs when only Valkyrien Skies, addons, and the mod I have specified are installed and no other mods

- [X] I have tested this issue and it occurs with only Valkyrien Skies, addons, and the mod I have specified

### Minecraft Version

1.20.x

### Mod Loader

Fabric

### Issue description

When using a Modern Beta preset with Valkyrian skies, attempting to assemble any physics object will generate additional junk blocks in addition. I have used a Eureka helm to demonstrate, but attempting to assemble using the builtin Valkyrian skies tools will do the same.

Issue appears to possibly be similar to: https://github.com/ValkyrienSkies/Valkyrien-Skies-2/issues/635.

![2024-02-01_12 28 55](https://github.com/ValkyrienSkies/Valkyrien-Skies-2/assets/154877329/66d23660-691f-4c25-8618-1cec46d970ee)
Unassembled ship (it is not touching the tree below it)
![2024-02-01_12 29 15](https://github.com/ValkyrienSkies/Valkyrien-Skies-2/assets/154877329/2ddaad61-ca1d-47a2-8c79-a6d119ec44db)

Farlands chunk being assembled as part of the ship.
![2024-02-01_12 29 31](https://github.com/ValkyrienSkies/Valkyrien-Skies-2/assets/154877329/e7f768ef-a1e5-4097-a62f-43fdfba0af66)

Zoomed out picture

This is the same Minecraft instance, but with the default generator selected.
![2024-02-01_13 13 41](https://github.com/ValkyrienSkies/Valkyrien-Skies-2/assets/154877329/ca796cc9-ccb0-401e-a10f-06b90ff2c785)
Unassembled ship
![2024-02-01_13 14 31](https://github.com/ValkyrienSkies/Valkyrien-Skies-2/assets/154877329/bcdaccc6-6d26-4611-a24c-6206e6b30229)

Assembled Ship

### Issue reproduction

Generate a world using any Modern Beta preset and then attempt to assemble any Valkyrian skies ship.

It is to note also: this issue also occurs in every version of modern beta and Valkyrian skies I have tested (1.20.1, 1.19.2 & 1.18.2).

Another note: this issue also (unsurprisingly) persists when using forge Valkyrian skies and Modern Beta through Sinytra connector, but the connector is NOT related to the issue.

### Logs

No logs, this is not a crash.

**Comment by PriestOfFerns** [comment]
It seems that modern beta causes chunks to generate in the ship yard. Which leads to ships being created with terrain chunks

**Comment by Zombiekris** [comment]
Would there be anyway to disable the far lands in modern beta or change where the ship yard is so this does not persist?

**Comment by NotaScout** [comment]
> Would there be anyway to disable the far lands in modern beta or change where the ship yard is so this does not persist?

No, it's not inherently in either mod without modifying either at a code level. I modified someone else's mod to essentially delete the far lands in the proto-chunk stage (I never published it), and it works well enough. Maybe I'll get around to that at some point.
